CHANGELOG.md

```diff
@@ -6,6 +6,112 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 <!--
 TODO: Insert version codename, and username of the contributor that named the release.
 -->
+## [23.02rc1] - 2023-02-08
+
+### Added
+
+ - Plugins: `sql` plugin command to perform server-side complex queries. ([#5679])
+ - JSON-RPC: new command `listpeerchannels` now contains information on direct channels with our peers. ([#5825])
+ - RPC `listpeers` output now has `num_channels`. ([#5968])
+ - JSON-RPC: `signinvoice` new command to sign BOLT11 ([#5697])
+ - JSON-RPC: `upgradewallet` command, sweeps all p2sh-wrapped outputs to a native segwit output ([#5670])
+ - JSON-RPC: `fundpsbt` option `nonwrapped` filters out p2sh wrapped inputs ([#5670])
+ - JSON-RPC: `listchannels` added a `direction` field (0 or 1) as per gossip specification. ([#5679])
+ - hsmd: New messages `hsmd_preapprove_keysend` and `hsmd_preapprove_invoice` added. ([#5821])
+ - JSON-RPC: New commands `preapprovekeysend` and `preapproveinvoice` added. ([#5821])
```

Comment:
Can we do new commands first?  And make format the same, e.g

    JSON-RPC: `signinvoice`: new command to sign BOLT11 ([#5697])